### PR TITLE
Create pull request for splitting previous commit

### PR DESCRIPTION
**Pull request by wufengyi**
Hi,merbanan

I have finished the splitting work for the commit you specified,
https://github.com/merbanan/rtl_433/pull/374

please review it and let me know the results.

Thanks.
Jeffson.

### CODE REVIEW

File: src/devices/oregon_scientific.c [hunk 1]
```diff
@@ -422,79 +419,67 @@ static int oregon_scientific_v3_parser(bitbuffer_t *bitbuffer) {
...
-	// 13..11 Current Speed, meters per second, LSD is 0.1 m/s
-	// 16..14 Average speed, meters per second, LSD is 0.1 m/s
-/*
-	int i;
-	printf("13..11 current speed, 16..14 average speed\n");
-	for (i=0;i<34;i+=2) printf("%-2d ", i/10); printf("\n");
-	for (i=0;i<34;i+=2) printf("%-2d ",i%10);printf("\n");
-	for (i=0;i<17;i++) printf("%02x ", msg[i]); printf("\n");
-*/
-        float gustWindspeed = (msg[5]&0x0f) /10.0F + ((msg[6]>>4)&0x0f) *1.0F + (msg[6]&0x0f) * 10.0F;
```
Comment (merbanan) [inline src/devices/oregon_scientific.c]: Does this compile ? gustWindspeed is used later in the code.

**Comment by merbanan** [comment]
Looks good, I will review in more detail later this day.

File: src/devices/oregon_scientific.c [hunk 2]
```diff
@@ -386,20 +394,19 @@ static int oregon_scientific_v3_parser(bitbuffer_t *bitbuffer) {
 
     if ((msg[0] == 0xf8) && (msg[1] == 0x24))    {
       if (validate_os_checksum(msg, 15) == 0) {
-        int  channel = get_os_channel(msg, ID_THGR810);
         float temp_c = get_os_temperature(msg, ID_THGR810);
         int humidity = get_os_humidity(msg, ID_THGR810);
-	int battery = get_os_battery(msg, ID_THGR810);
```
Comment (merbanan) [inline src/devices/oregon_scientific.c]: Is this correct, channel and battery is removed ?

**Comment by merbanan** [comment]
Ok, there was only that one thing I could see. The other patch still has that line left. You can do a diff with kdiff3, meld or xxdiff to see if you missed some other line.

**Comment by wufengyi** [comment]
I have fixed the code,please review again.
thanks.

**Pull request merged**